src/Behavioral.Automation/Services/BrowserRunner.cs

```diff
@@ -2,15 +2,16 @@
 using System.Collections.Generic;
 using JetBrains.Annotations;
 using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Chromium;
 using OpenQA.Selenium.Remote;
 
 namespace Behavioral.Automation.Services
 {
     public class BrowserRunner
     {
-        public RemoteWebDriver Driver;
```

Comment:
Why renamed? We can use not only ChromiumDriver.

Reply:
Maybe, we can use an interface here

Reply:
Changed to IWebDriver since RemoteWebDriver cannot be cast into ChromeDriver